packages/core/src/app.d.ts

```diff
@@ -9,7 +9,3 @@ declare namespace App {
   // interface Session {}
   // interface Stuff {}
 }
-
-declare module '*.svelte' {
```

Comment:
As far as I can tell, this isn't needed for typechecking. I might be missing context about why it was added, but if we can remove it then migration to svelte 5 will be just a little simpler 🎉

Reply:
I think this was before the types around `SvelteComponent` stabilized, so good catch 👍